linked_list/linked_list.py

```diff
@@ -41,13 +41,30 @@ def add_to_tail(self, value):
 
     def remove_head(self):
         pass
-# if self.head = true then
-# removed_head= LinkedList.head
-# LinkedList.head.remove()
-# return removed_head
+
+        # self.head and next
+# if self.head = true
+# new_head= self.head.next_node
+# self.head.remove()
+# return self.head=new_head
+# self.length -=1
 
     def contains(self, value):
-        pass
+        current = self.head
+        if current == value:
+            return True
+        elif current:
+            current = current.next_node
+        else:
+            return False
 
     def get_max(self):
```

Comment:
Naaz your code looks great and one feedback I have would be use a meaningful variable name. For example, for finding max, instead of total maxValue will make reading your code easy, one has to pause and think what total is.